### ISSUE TITLE
Is the full version coming out or was it abandoned?

### ISSUE THREAD

**Issue by durrburger93** [notfromstatefarm/nvflash#15]
The description still says the GUI version is coming by the end of August. Is this still coming or was there a change in plans?

**Comment by Gtx970user** [comment]
I think it was abandoned tbh

**Comment by Tinnci** [comment]
August, 2024, perhaps?

**Comment by sternskr** [comment]
> August, 2024, perhaps?

Yeah I'm starting to think it was abandoned. I've been following for a while. Even if it takes until 2024 I'll still be happy to have a tool that can do what the GUI is being pitched to accomplish

**Comment by ddkcstream** [comment]
是被放弃了吗？是否有计划开源？
